benchmark/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
copy from android-architecture-components, there should be more

Reply:
:+1: